docs/concepts.rst

```diff
@@ -554,6 +554,13 @@ Some other tips when using SubDAGs:
    effectively limit its parallelism to one. Using LocalExecutor can be
    problematic as it may over-subscribe your worker, running multiple tasks in
    a single slot
+-  do not create more SUBDags then your concurrency limit or the scheduler
```

Comment:
The capitalization is all over the place in this doc, let's stick with "SubDag" since that is what is in code.